source/mainnet/smart-contracts/tutorials/voting/voting-dapp.rst

```diff
@@ -0,0 +1,296 @@
+
+    The |bw| is in active development.
+
+You can choose from two different workflows that will guide you through the setup steps. Workflow 1 is easier.
+The server that the |bw| connects to is hosted by Concordium in workflow 1. This setup will take care of the
+Concordium node on your behalf. Workflow 2 is for
+advanced readers that want to build all the components from the source code and connect the |bw|
+to their own hosted Concordium node.
+
+.. dropdown:: Workflow 1 - Setup wallet connected to hosted node (click here)
+
+    You are ready now to install the |bw|. Install the extension for your browser as described :ref:`here<setup-browser-wallet>`.
+
+    The |bw| connects to a server hosted by Concordium which will take care of the Concordium node on your behalf.
+
+    You are ready now to start the |bw| by clicking on the Concordium icon at the top right of the
+    ``Chrome`` browser.
+
+    .. image:: ./images/wCCD_tutorial_18.png
+        :width: 100 %
+
+    .. note::
+
+        The puzzle icon at the top right of the ``Chrome`` browser allows you to manage your browser extensions.
+        You can enable pinning of the |bw|.
+
+        .. image:: ./images/wCCD_tutorial_13.png
+            :width: 30 %
+
+    Create a new account on testnet by going through the setup steps of the |bw|.
+    You have to choose a password to secure your |bw|.
+    This password is needed to log in to your |bw|.
+
+    .. image:: ./images/wCCD_tutorial_15.png
+        :width: 30 %
+
+    The |bw| creates a unique secret recovery phrase. Write down the secret recovery phrase
+    and keep it in a safe place to be able to recover your accounts in case
+    you lose access to your device.
+
+    You have completed the setup. Check that your |bw| is connected to the testnet.
+
+    .. image:: ./images/wCCD_tutorial_16.png
+        :width: 30 %
+
+    .. image:: ./images/wCCD_tutorial_17.png
+        :width: 30 %
+
+    Before you can create a new account. You need to create an identity.
+
+    .. image:: ./images/wCCD_tutorial_19.png
+        :width: 30 %
+
+    .. image:: ./images/wCCD_tutorial_20.png
+        :width: 30 %
+
+    .. image:: ./images/wCCD_tutorial_21.png
+        :width: 30 %
+
+    .. image:: ./images/wCCD_tutorial_22.png
+        :width: 30 %
+
+    You are ready now to create a new account on testnet.
+
+    .. image:: ./images/wCCD_tutorial_19.png
+        :width: 30 %
+
+    .. image:: ./images/wCCD_tutorial_20.png
+        :width: 30 %
+
+    You completed the |bw| setup. Send some CCD to your new account or request some CCD from the testnet faucet button within the |bw|.
+    Check that your account balance is displayed and you have enough
+    CCD to be able to execute transactions.
+
+    .. note::
+        You are connected to a website with your |bw| when you see the green ``Connected`` button.
+        You can toggle on/off the connection by clicking on the button.
+
+    .. image:: ./images/wCCD_tutorial_14.png
```

Comment:
If this is the picture from the `wCCD` tutorial then we should link to that picture in the repo so we don't duplicate or if we want to copy the picture into this repo then I would rename it from `wCCD` to `voting`.

Reply:
There are a few pictures that are not used, so we can remove them from the folder completely.